website/assets/styles/pages/query-detail.less

```diff
@@ -1,5 +1,35 @@
 #query-detail {
 
-  // …
+  h5 {
+    font-size: 18px;
+    font-weight: bold;
+  }
+
+  h6 {
+    font-size: 16px;
+    line-height: 24px;
+  }
+
+  a {
+    font-size: 18px;
+    color: @core-vibrant-blue;
+  }
+
+  .query-tip {
+    background-color: @ui-off-white;
+
+    #lightbulb {
```

Comment:
could you swap this `id` out for something else?

background: js can break when an id is used more than once on the same page, so by convention, we try to avoid using ids in parasails apps except for page wrappers -- or in special cases where no other selector would work because some plugin requires you to use an id